playground/backend/cmd/server/controller.go

```diff
@@ -141,35 +142,55 @@ func (controller *playgroundController) GetCompileOutput(ctx context.Context, in
 
 //GetListOfExamples returns the list of examples
 func (controller *playgroundController) GetListOfExamples(ctx context.Context, info *pb.GetListOfExamplesRequest) (*pb.GetListOfExamplesResponse, error) {
-	// TODO implement this method
-	example1 := pb.Example{ExampleUuid: "001", Name: "Example1", Description: "Test example 1", Type: pb.ExampleType_EXAMPLE_TYPE_DEFAULT}
-	example2 := pb.Example{ExampleUuid: "003", Name: "Example3", Description: "Test example 3", Type: pb.ExampleType_EXAMPLE_TYPE_KATA}
-
-	cat1 := pb.Categories_Category{
-		CategoryName: "Common",
-		Examples:     []*pb.Example{&example1, {ExampleUuid: "002", Name: "Example2", Description: "Test example 1", Type: pb.ExampleType_EXAMPLE_TYPE_UNIT_TEST}},
+	cd := storage.New()
+	examples, err := cd.GetListOfExamples(ctx, info.Sdk, info.Category)
+	if err != nil {
+		logger.Errorf("%s: GetListOfExamples(): cloud storage error: %s", err.Error())
+		return nil, errors.InternalError("GetListOfExamples(): ", err.Error())
 	}
-	cat2 := pb.Categories_Category{
-		CategoryName: "I/O",
-		Examples:     []*pb.Example{&example2},
+	response := pb.GetListOfExamplesResponse{SdkExamples: make([]*pb.Categories, 0)}
+	for sdk, categories := range *examples {
+		sdkCategories := pb.Categories{Sdk: pb.Sdk(pb.Sdk_value[sdk]), Categories: make([]*pb.Categories_Category, 0)}
+		for categoryName, examplesArr := range categories {
+			getCategoryWithExamples(categoryName, examplesArr, &sdkCategories)
+		}
+		response.SdkExamples = append(response.SdkExamples, &sdkCategories)
 	}
-	javaCats := pb.Categories{Sdk: pb.Sdk_SDK_JAVA, Categories: []*pb.Categories_Category{&cat1, &cat2}}
-	goCats := pb.Categories{Sdk: pb.Sdk_SDK_GO, Categories: []*pb.Categories_Category{&cat1, &cat2}}
-	response := pb.GetListOfExamplesResponse{SdkExamples: []*pb.Categories{&javaCats, &goCats}}
 	return &response, nil
 }
 
+func getCategoryWithExamples(categoryName string, examplesArr storage.ExamplesInfo, sdkCategories *pb.Categories) {
```

Comment:
```suggestion
func getCategoryWithExamples(categoryName string, examples storage.ExamplesInfo, sdkCategories *pb.Categories) {
```

Reply:
Maybe we could move this method to some other file? Let's place here (`controller.go`) API's methods only.